test/config.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const home = require('os').homedir()
+
+
```

Comment:
is this double line break intentional?

Reply:
Yes, to separate `require`s from code, but it is not essential in this case. Removed in commit e7e672e.